pipeline_dp/dp_computations.py

```diff
@@ -103,50 +104,70 @@ def compute_sigma(eps: float, delta: float, l2_sensitivity: float):
         delta: The delta value.
         l2_sensitivity: The L2 sensitivity.
     """
-    # TODO: use named arguments, when argument names are added in PyDP on PR
-    # https://github.com/OpenMined/PyDP/pull/398.
-    return dp_mechanisms.GaussianMechanism(eps, delta, l2_sensitivity).std
+    return dp_mechanisms.GaussianMechanism(epsilon=eps,
+                                           delta=delta,
+                                           sensitivity=l2_sensitivity).std
 
 
-def apply_laplace_mechanism(value: float, eps: float, l1_sensitivity: float):
+def apply_laplace_mechanism(value: float,
+                            eps: float,
+                            l1_sensitivity: float,
+                            noise_standard_deviation: Optional[float] = None):
     """Applies the Laplace mechanism to the value.
+       If noise_standard_deviation is set, it is used and eps is ignored.
 
     Args:
         value: The initial value.
         eps: The epsilon value.
         l1_sensitivity: The L1 sensitivity.
+        noise_standard_deviation: The standard deviation for the noise.
 
     Returns:
         The value resulted after adding the noise.
     """
-    mechanism = dp_mechanisms.LaplaceMechanism(epsilon=eps,
-                                               sensitivity=l1_sensitivity)
+    if noise_standard_deviation is not None:
+        mechanism = dp_mechanisms.LaplaceMechanism(epsilon=1 /
```

Comment:
Please add comment that here workaround is used, since we can't set the Laplace parameter directly